kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -69,39 +69,25 @@ export async function getPortableExperience(pid: string): Promise<PortableExperi
   }
 }
 
-async function parsePortableExperienceUrn(sceneUrn: string): Promise<DecentralandAssetIdentifier> {
-  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(sceneUrn)
-  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
-    throw new Error(`Could not parse portable experience from urn: ${sceneUrn}`)
-  }
-  return parsedUrn
-}
-
-function isPortableExperience(dclId: DecentralandAssetIdentifier): dclId is OffChainAsset {
-  /* tslint:disable-next-line */
-  return !!(dclId as OffChainAsset).registry && (dclId as OffChainAsset).registry === 'static-portable-experiences'
-}
-
 export async function getPortableExperienceFromS3Bucket(sceneUrn: string) {
-  const parsedUrn: DecentralandAssetIdentifier = await parsePortableExperienceUrn(sceneUrn)
-  const mappingsUrl = await resolveContentUrl(parsedUrn)
+  const mappingsUrl = await resolveUrlFromUrn(sceneUrn)
   if (mappingsUrl === null) {
     throw new Error(`Could not resolve mappings for scene: ${sceneUrn}`)
   }
-  const baseUrl: string = new URL('..', mappingsUrl).toString()
-  const mappingsFetch = await fetch(`${baseUrl}mappings`)
+  const mappingsFetch = await fetch(mappingsUrl)
   const mappingsResponse = (await mappingsFetch.json()) as MappingsResponse
 
   const sceneJsonMapping = mappingsResponse.contents.find(($) => $.file === 'scene.json')
 
   if (sceneJsonMapping) {
+    const baseUrl: string = mappingsUrl.slice(0, -1 * 'mappings'.length)
```

Comment:
Why did you drop the url resolution? that approach sounds hackish and assumes it always returns a url ending in "mappings"

```suggestion
    const baseUrl: string = `${new URL('..', mappingsUrl)}`
```

Reply:
It is a hack that I don't expect to last, I was just testing. Anyway, the problem I faced was that the resolver returns only: `https://static-pe.decentraland.io/${asset.id}/mappings`, so if I slice only the baseUrl, then I'll have to manually add the assetId. Any idea?

Reply:
I don't understand the problem, is it related to this line?

Reply:
Yes, the solution that I found using the url resolution is something like:


```
const baseUrl: string = `${new URL('..', mappingsUrl)}`
const sceneUrl: string = `${baseUrl}/${portableExperienceId}/${sceneHash}`
```

I don't really love it, what do you think?

Reply:
Oh I understand, I think it is easier than it looks since the baseUrl already contains the portableExperienceId

```ts
// this resolves .. from the /mappings url
const baseUrl: string = new URL('..', mappingsUrl).toString()

// this resolves the scene.json
const sceneUrl: string = new URL(sceneHash, baseUrl).toString()
```

Reply:
No, it doesn't :( the value for `sceneUrl` after running that is `https://static-pe.decentraland.io/scene.json`

Reply:
Omg, something is wrong, thanks for clarifying. I will check the library

Reply:
I checked, it was my mistake, I accidentally added two dots instead of one in the URL('..')

This one should work

```ts
// this resolves .. from the /mappings url
const baseUrl: string = new URL('.', mappingsUrl).toString()

// this resolves the scene.json
const sceneUrl: string = new URL(sceneHash, baseUrl).toString()
// this also works, and it sounds better from a security point of view
const sceneUrl: string = `${baseUrl}/${sceneHash}`
```

```js
Welcome to Node.js v14.15.0.
Type ".help" for more information.
> r = require('@dcl/urn-resolver')
{
  LandUtils: [Getter],
  resolveContentUrl: [Getter],
  parseUrn: [AsyncFunction: parseUrn],
  resolveUrlFromUrn: [AsyncFunction: resolveUrlFromUrn]
}
> r.resolveUrlFromUrn('urn:decentraland:off-chain:static-portable-experiences:quest-1').then($ => console.log(new URL('aHash', new URL('.', $)).toString()))
Promise { <pending> }
> https://static-pe.decentraland.io/quest-1/aHash
```

Reply:
Ohh, thanks!